scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/solver/PrioritySolver.java

```diff
@@ -116,6 +116,7 @@ public Optional<Plan> getNextSolution() {
       try {
         initializePlan();
         if(problem.getInitialSimulationResults().isPresent()) {
+          logger.info("Loading initial simulation results from the DB");
```

Comment:
Minor, can ignore: this is one I especially don't feel strongly about one way or the other. I'm just uncertain if a non-debugging user cares if we're loading sim results from the DB.
```suggestion
          logger.debug("Loading initial simulation results from the DB");
```